src/components/Editor/Repeat/RepeatSummary.vue

```diff
@@ -92,6 +92,15 @@ export default {
 		},
 	},
 	computed: {
+		/**
+		 * Returns whether or not to display the summary.
+		 * We do not want to show it if it doesn't repeat
+		 *
+		 * @returns {boolean}
+		 */
+		display() {
+			return this.frequency !== 'NONE'
+		},
 		summary() {
 			if (this.frequency === 'NONE') {
 				return this.$t('calendar', 'Does not repeat')
```

Comment:
So…this isn't shown anymore then? Might as well remove it.

Reply:
Indeed. Let me remove that

Reply:
Actually no, we need this.

The RepeatSummaryReadOnly component extends RepeatSummary: https://github.com/nextcloud/calendar/blob/master/src/components/Editor/Repeat/RepeatSummaryReadOnly.vue#L37

And this is also shown when the repeat is non-recurring.

Reply:
Please add a comment so this isn't forgotten, then. :+1: